code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -1174,6 +1127,37 @@
 		return TRUE
 	return FALSE
 
+
+
+/////////////////////////////////////
+////////  Atmospheric stuff  ////////
+/////////////////////////////////////
+
+/obj/vehicle/sealed/mecha/remove_air(amount)
+	if(use_internal_tank)
+		return cabin_air.remove(amount)
+	return ..()
+
+/obj/vehicle/sealed/mecha/return_air()
+	if(use_internal_tank)
+		return cabin_air
+	return ..()
+
+/obj/vehicle/sealed/mecha/return_analyzable_air()
+	return cabin_air
+
+///fetches pressure of the gas mixture we are using
+/obj/vehicle/sealed/mecha/proc/return_pressure()
+	var/datum/gas_mixture/t_air = return_air()
+	return t_air?.return_pressure()
```

Comment:
Just call it air, you're not "guaranteed" it's turf air, and this var name is ugly

Reply:
Change done